tests/unit/webscraper_io_client_test.py

```diff
@@ -0,0 +1,225 @@
+"""Unit Tests for the Webscraper.io client class."""
+
+from pathlib import Path
+from typing import Any
+from unittest.mock import AsyncMock, MagicMock
+
+import pytest
+from httpx import AsyncClient, HTTPStatusError, RequestError, Response
+
+from cpeq_infolettre_automatique.webscraper_io_client import WebScraperIoClient
+
+
+# mypy: disable-error-code="method-assign"
+
+sitemaps: list[dict[str, str]] = [
```

Comment:
Si on connait le format des sitemaps on pourrait utiliser un Pydantic BaseModel pour faire la valdiation

Pour les types de Network: https://docs.pydantic.dev/latest/api/networks/

```suggestion
from pydantic import BaseModel, HttpUrl

class Sitemap(BaseModel):
    id: str
    name: str
    url: HttpUrl
```

Reply:
Done